### ISSUE TITLE
Add <IPPH> and <APPH> (if necessary) to deal with IPA + Greek unification

### ISSUE THREAD

**Issue by jungshik** [googlei18n/noto-fonts#489]
Spun off from #440. 

Below is @tiroj's comment on how to deal with Greek and IPA unification in the Unicode:

As noted in bug 440, four characters from the Unicode Greek block are unified with IPA letters. 
This unification has always seemed to me a bad idea: IPA is more conveniently handled uniformly 
as an extension of the Latin script, and the style of Greek glyphs in a typeface may not harmonise 
sufficiently with the Latin for this dual rôle. It would have been better to encode Latin beta, theta, 
lambda and chi letters, as was done with the Latin gamma ɣ and iota ɩ. In the case of Noto Serif, 
the design of the Greek β,θ and χ is such that they harmonise reasonably well among Latin letters; 
the λ is, as usual, the one that stands out most.

In the Brill types, there are variant glyphs for these Greek characters, designed to harmonise 
better with Latin forms. These are mapped in the <locl> feature for the <IPPH> language system 
tag. [Peter Constable at Microsoft registered this language system tag for IPA and <APPH> for 
Americanist phonetic notation, following discussions about the best level at which to address 
glyph variants for these writing systems. IPA functions like a script, in which it is possible to write 
any language, but from an OTL script itemisation perspective it is a subset of Latin; hence the 
distinction needs to be made at the language system level.] This could be considered for Noto Serif, 
to provide for more fully harmonised Latin forms of these letters.

![image](https://cloud.githubusercontent.com/assets/8578343/9587331/e89e5bf6-4fd6-11e5-8a9b-52db957ec051.png)

In Noto Sans the Greek letters work pretty well with the Latin, although the β is massively wide:

![image](https://cloud.githubusercontent.com/assets/8578343/9587347/fcf32938-4fd6-11e5-93f5-8278604cb1df.png)

@waksmonskiMT : could you review @tiroj's comments with your team? 

/cc @fontguy @roozbehp @xiangyexiao @waksmonskiMT @dougfelt

**Comment by moyogo** [comment]
Comment on #440 that should be here instead:
Note that ꞵ U+A7B5 and ꭓ U+AB53 are Latin small letter beta and Latin small
letter chi. Latin beta was added for orthographic support and Latin chi for
German dialect phonetic transcription but is also used in orthographies.
Both are glyphically appropriate for IPA, meaning the IPA IPPH forms of the
Greek letters can look the same (or the other way around).

**Comment by SteveMatteson** [comment]
I do not believe any work needs to be done here. Latin and Greek are harmonized so 'more Latin or more Greek' goes against the idea of a harmonized set of alphabets.

**Comment by marekjez86** [comment]
from MTI: "Tom added anchors to Greek beta, theta, lambda and chi. Fix anchors in beta. Adding IPPH and APPH to locl feature is an enhancement and not a bug."
